browser-client.cpp

```diff
@@ -415,7 +427,11 @@ void BrowserClient::OnAcceleratedPaint(CefRefPtr<CefBrowser>,
 	UpdateExtraTexture();
 	obs_leave_graphics();
 
+#if CHROME_VERSION_BUILD >= 6367
```

Comment:
Is this a theoretical or confirmed problem? If this is definitely creating an observable problem after building against the latest version of CEF, then we should address it now. If this is merely a theoretical problem that has not been confirmed, then I don't want to spend too much time dwelling on it. I'd rather make it possible to make builds against CEF 122/124+ and figure out what problems exist and fix them.

Reply:
Although IOSurface pointers could be valid for this as discussed with @PatTheMav , DXGI handles are not. I think we should just remove this line and remove `last_handle`. Re-importing every frame just cost 50~100us and I think we can omit it.

The problem is this is a real theoretical problem and we will not know if underlying chromium code changes behaviour. The suggested behaviour doesn't let us "cache".

Considering a frame skip could be rare and hard to notice, I think we should just disable such "optimization" and do the absolute right thing, prevent letting someone in the world wondering why frames are skipped.

Reply:
A problem can't be both "real" and "theoretical". I'm asking if there's an observable failure/error when building with this code against CEF 124+. As it is right now, we cannot test obs-browser with CEF 124+ in any meaningful way without this PR, which is my concern at the moment. I'm not saying "don't ever fix this". I'm saying "does this need to be fixed before merging", because otherwise we can't meaningfully test CEF 124+ without specifically pulling this PR locally.

Reply:
Theoretically, yes. You can also anaylze the code and get the same conclusion. So I think we would better remove this >= 6367 to take no risk.

Reply:
For now, I've left the code to set bs->last_handle but it's not read. I personally didn't see any strange behaviours either way, but would rather go the safer route for now.